packages/console/src/pages/CloudPreview/pages/About/options.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Option as SelectorOption } from '@/pages/CloudPreview/components/CardSelector';
+
+import { CompanySize, Reason, Title } from '../../types';
+
+export const titleOptions: SelectorOption[] = [
+  { title: 'cloud_preview.about.title_developer', value: Title.Developer },
+  { title: 'cloud_preview.about.title_team_lead', value: Title.TeamLead },
+  { title: 'cloud_preview.about.title_ceo', value: Title.Ceo },
+  { title: 'cloud_preview.about.title_cto', value: Title.Cto },
+  { title: 'cloud_preview.about.title_product', value: Title.Product },
+  { title: 'cloud_preview.about.title_others', value: Title.Others },
```

Comment:
nit: you can put these in a nested object called `title`, so the structure will be better organized. For example in the phrase json you can use:

```
title: {
  developer: 'Developer',
  team_lead: 'Team Lead',
  ...,
}
```

Reply:
Add an issue to track this.
LOG-5419